pkgs/applications/networking/ouroboros/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv
+, lib
+, cmake
+, dnsutils
+, fetchgit
+, fuse
+, libgcrypt
+, openssl
+, pkg-config
+, protobuf
+, protobufc
+, systemd
+}:
+stdenv.mkDerivation rec {
+  pname = "ouroboros";
+  version = "0.20.1";
+  src = fetchgit {
+    url = "git://ouroboros.rocks/${pname}";
+    rev = version;
+    hash = "sha256-r5IyWEdNDfrJnW8Th+pWvGVFHsxEp91IQomesLNADJI=";
+  };
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+  ];
+  buildInputs = [
+    protobufc
+    dnsutils
+    fuse
+    libgcrypt
+    openssl
+    protobuf
+    systemd
```

Comment:
You marked `meta.platforms` as `unix`, but `systemd` is `linux`-only. Won't build on non-Linux, as OfBorg shows:
```
Cannot nix-instantiate `ouroboros' because:
error:
       … while evaluating the attribute 'buildInputs' of the derivation 'ouroboros-0.20.1'
[...]
       error: Package ‘systemd-254.3’ in /private/var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/builder/macstadium-x86-44911362/pkgs/os-specific/linux/systemd/default.nix:769 is not available on the requested hostPlatform:
         hostPlatform.config = "x86_64-apple-darwin"
```

Not sure if you want to bother with getting it working (and keeping it working) on Darwin, LMK what you think.